kolibri/core/assets/src/core_app_instance.js

```diff
@@ -3,6 +3,7 @@
 require('normalize.css');
 require('./font-NotoSans.css');
 require('./core-global.styl');
+require('babel-polyfill');
```

Comment:
We weren't loading in the babel polyfill anywhere, so things like Symbols were not being properly polyfilled.